source/guide_cryptpad.rst

```diff
@@ -0,0 +1,173 @@
+.. author:: humbug <uberspace@humbug.pw>
+.. highlight:: console
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/cryptpad.png
+      :align: center
+
+########
+Cryptpad
+########
+
+`Cryptpad`_ is a Zero Knowledge realtime collaborative editor. It is based on :manual:`Node.js <lang-nodejs>` and comes with encryption. It relies on the `ChainPad`_.
+
+----
+
+.. note:: For this guide you need some tools:
+
+  * `Web Backends`_ (You have to opt-in the Beta. Write an E-Mail to `hallo@uberspace.de`_
+  * :manual:`Node.js <lang-nodejs>` and its package manager :manual_anchor:`npm <lang-nodejs.html#npm>`
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`domains <web-domains>`
+
+Prerequisites
+=============
+
+After we're enrolled into the beta, we need to setup the Web Backends:
+
+::
+
+  [isabell@stardust ~]$ uberspace web backend set isabell.uber.space --http --port 3000
+  Set backend for isabell.uber.space/ to port 3000; please make sure something is listening!
+  You can always check the status of your backend using "uberspace web backend list".
+  [isabell@stardust ~]$
+
+Now let's get started with Cryptpad.
+
+We're using :manual:`Node.js <lang-nodejs>` in the stable version 6:
```

Comment:
Node 6 will reach its EOL next month, which means we will remove it from U7 hosts. I just built Cryptpad with Node 10 and it seems to work just fine …

Reply:
Alright